addons/sale_stock/sale_stock.py

```diff
@@ -271,6 +271,20 @@ def _run_move_create(self):
             vals.update({'sequence': self.sale_line_id.sequence})
         return vals
 
+    @api.cr_uid_context
+    def create(self, cr, uid, vals, context=None):
```

Comment:
Why writing it old api ? We do not need any mapping here.

Why not use @api.model ?